helpers/helpers_emba_status_bar.sh

```diff
@@ -0,0 +1,412 @@
+#!/bin/bash
+
+# EMBA - EMBEDDED LINUX ANALYZER
+#
+# Copyright 2022 Siemens AG
+#
+# EMBA comes with ABSOLUTELY NO WARRANTY. This is free software, and you are
+# welcome to redistribute it under the terms of the GNU General Public License.
+# See LICENSE file for usage of this software.
+#
+# EMBA is licensed under GPLv3
+#
+# Author(s): Pascal Eckmann
+
+# Description: Show stats about EMBA run on the bottom of the terminal window
+
+
+# PID for box updater threads
+export PID_SYSTEM_LOAD=""
+export PID_STATUS=""
+export PID_MODULES=""
+export PID_STATUS_2=""
+# Path to status tmp file
+# each line is dedicated to a specific function
+# 1: Count of boxes visible
+# 2: CPU load string (needs to be cached, because it takes about a second to get the information)
+# 3: Start time of status bar - not exactly the same as the EMBA timer, but near enough to get an idea
+# 4: Count modules
+export STATUS_TMP_PATH=""
+
+# helper for box drawing
+repeat_char(){
+  local REP_CHAR="$1"
```

Comment:
Shouldn't we use "${1:-}" in function parameters?

Reply:
yes, changed it